src/stores/__tests__/chatExtras.spec.js

```diff
@@ -161,4 +162,24 @@ describe('chatExtrasStore', () => {
 			expect(chatExtrasStore.chatInput['token-1']).not.toBeDefined()
 		})
 	})
+
+	describe('text parsing', () => {
```

Comment:
You should test functions directly with textParse.spec.js. We could add more examples and tests there later.

In this file, you can call setChatInput() with `{user}` and test that getChatInput() returns `@user`